asset-registry/src/lib.rs

```diff
@@ -14,7 +14,7 @@ use sp_runtime::{
 	DispatchResult,
 };
 use sp_std::prelude::*;
-use xcm::{v2::prelude::*, VersionedMultiLocation};
+use xcm::{latest::prelude::*, VersionedMultiLocation};
```

Comment:
we intentionally specify a version here to make it obvious and avoid unexpected version change

Reply:
also this have MultiLocation in storages so need to make sure it remains the same, otherwise we need a migration

Reply:
`MultiLocation` encoding changed from v2 to v3 ([test](https://github.com/open-web3-stack/open-runtime-module-library/pull/886/commits/b142f569c3d65827859afeb70f5f37eee9c2a68d)) and a migration is needed. In that regard, which migration strategy did you followed in the past for this pallet if any?

Reply:
do we have to change? can we just keep using v2? v3 MultiLocation can be downgraded to v2 right?

Reply:
we can have a migration function translate the storages. I don’t expect it will have many keys so a one off migration should be fine

Reply:
If we keep V2 in this pallet we need to probably update its associated types implementors to handle the downgrades. I think the easiest path is just move to V3 and add the migration function.